app.py

```diff
@@ -180,6 +180,8 @@
         "SERVER_PORT": "8086",
         "SPRING_CLOUD_CONFIG_URI": "http://openchallenges-config-server:8090",
         "SERVICE_REGISTRY_URL": "http://openchallenges-service-registry:8081/eureka",
+        "OPENCHALLENGES_IMAGE_SERVICE_THUMBOR_HOST": "https://{fully_qualified_domain_name}/img/",
```

Comment:
this needs to be an fstring.. `f"https://{fully_qualified_domain_name}/img/"`

Reply:
Fixed